test/test_heuristics.rb

```diff
@@ -127,6 +127,14 @@ def test_fs_by_heuristics
     })
   end
 
+  # Candidate languages = ["Genie", "GLSL", "Gosu", "JavaScript"]
+  def test_gs_by_heuristics
+    assert_heuristics({
+      "GLSL" => all_fixtures("GLSL", "*.gs"),
+      "Gosu" => all_fixtures("Gosu", "*.gs"),
```

Comment:
Could we also check here that no Genie or JavaScript files are classified as GLSL or Gosu by our heuristic rules?